File: 001/Problem1.py
import time


def problem001():
    su = 0
    for i in range(1, 1000):
        if i % 3 == 0 or i % 5 == 0:
            su += i
    print(su)


start = time.time()
for i in range(1, 10000):
    problem001()

end = time.time()
print("Function took " + str(end - start))

File: 001/Problem1.go
package main

import (
	"fmt"
	"time"
)

func problem001() {
	var sum int = 0
	for i := 1; i < 1000; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}
	fmt.Println(sum)
}

func main() {
	start := time.Now()
	for i := 0; i < 10000; i++ {
		problem001()
	}
	elapsed := time.Since(start)
	fmt.Printf("Function took %s", elapsed)
}
